src/pages/home/report/ReportActionsView.js

```diff
@@ -33,7 +33,7 @@ const propTypes = {
     report: reportPropTypes.isRequired,
 
     /** Array of report actions for this report */
-    reportActions: PropTypes.objectOf(PropTypes.shape(reportActionPropTypes)),
+    reportActions: PropTypes.objectOf(PropTypes.shape(reportActionPropTypes)).isRequired,
```

Comment:
Actually you're right. The doc said to do this and I thought it was necessary for how we're computing the `reportActions` so that they wouldn't be empty but that actually gets handled just fine.